### PR TITLE
style: capitalization

### PR DESCRIPTION
**Pull request by wky0211**
OCD friendly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -309,7 +309,7 @@ Logo icons created by Freepik - Flaticon
 
 [![My idlers screenshot3](https://cdn.write.corbpie.com/wp-content/uploads/2022/03/My-idlers-v2-servers-home.jpg)](https://cdn.write.corbpie.com/wp-content/uploads/2022/03/My-idlers-v2-servers-home.jpg)
 
-[![My idlers screenshot4](https://cdn.write.corbpie.com/wp-content/uploads/2022/03/My-idlers-v2-YABs.jpg)](https://cdn.write.corbpie.com/wp-content/uploads/2022/03/My-idlers-v2-YABs.jpg)
+[![My idlers screenshot4](https://cdn.write.corbpie.com/wp-content/uploads/2022/03/My-idlers-v2-YABS.jpg)](https://cdn.write.corbpie.com/wp-content/uploads/2022/03/My-idlers-v2-YABS.jpg)
```
Comment (cp6) [inline README.md:312]: The URLs will have to stay as is

**Review by cp6 — CHANGES_REQUESTED** [review]
All good except the URL will have to stay as original

File: README.md (hunk 1 above)
Comment (wky0211) [reply README.md:312]: Oops, forget that URLs are case sensitive.

Comment (wky0211) [reply README.md]: Fixed!

**Comment by cp6** [comment]
Thanks!

**Pull request merged**